lighty-examples/lighty-community-aaa-restconf-app/src/main/java/io/lighty/kit/examples/community/aaa/restconf/Main.java

```diff
@@ -136,26 +136,11 @@ private void startLighty(final ControllerConfiguration controllerConfiguration,
         this.restconf.startServer();
     }
 
-    @SuppressWarnings("IllegalCatch")
-    private void closeLightyModule(final LightyModule module) {
-        if (module != null) {
-            try {
-                module.shutdown().get(modulesConfig.getModuleTimeoutSeconds(), TimeUnit.SECONDS);
-            } catch (final Exception e) {
-                LOG.error("Exception while shutting down {} module: ", module.getClass().getSimpleName(), e);
-                if (e instanceof InterruptedException) {
-                    Thread.currentThread().interrupt();
-                }
-            }
-        }
-    }
-
     public void shutdown() {
         LOG.info("Lighty.io, Restconf and AAA module shutting down ...");
         final Stopwatch stopwatch = Stopwatch.createStarted();
-        closeLightyModule(this.aaaLighty);
-        closeLightyModule(this.restconf);
-        closeLightyModule(this.lightyController);
+        LightyModuleUtils.stopMultipleModules(this.modulesConfig.getModuleTimeoutSeconds(), TimeUnit.SECONDS,
```

Comment:
No, do not add `stopMultipleModules` - you are having trouble to correctly track individual shutdowns